config/serverless.chat.yml

```diff
@@ -23,4 +23,7 @@ xpack.wciExternalServer.enabled: true
 xpack.spaces.maxSpaces: 1
 
 ## Content Connectors in stack management
-xpack.contentConnectors.enabled: false
\ No newline at end of file
+xpack.contentConnectors.enabled: false
+
+## Disable Kibana Product Intercept
+xpack.product_intercept.enabled: false
```

Comment:
Can you please help me understand why intercepts are disabled for the chat project type specifically and for all other projects types it's enabled by default?

Reply:
It's really because it's still in development, there's no requirement to enable it for chat as of now